src/Symfony/Bundle/FrameworkBundle/Resources/config/translation.xml

```diff
@@ -17,9 +17,8 @@
             <call method="setConfigCacheFactory">
                 <argument type="service" id="config_cache_factory" />
             </call>
-
-            <autowiring-type>Symfony\Component\Translation\TranslatorInterface</autowiring-type>
         </service>
+        <service id="Symfony\Component\Translation\TranslatorInterface" alias="translator.default" />
```

Comment:
the alias should be ``translator`` IMO instead, so that autowiring injects the real translator when using decoration. Otherwise, we loose the profiler integration when using autowiring and someone decorates the main ``translator`` alias rather than the private service.

And this alias should also be private (as any alias created purely for autowiring purpose anyway)